augur/frequencies.py

```diff
@@ -97,6 +101,11 @@ def run(args):
     columns_to_load = [metadata_object.id_column, METADATA_DATE_COLUMN]
     if args.weights_attribute:
         columns_to_load.append(args.weights_attribute)
+
+    use_region_column = any(region != DEFAULT_REGION for region in args.regions)
+    if use_region_column:
+        columns_to_load.append(REGION_COLUMN)
+
```

Comment:
Suggestion: rename variable to reflect filtering behavior

```suggestion
    filter_to_region = any(region != DEFAULT_REGION for region in args.regions)
    if filter_to_region:
        columns_to_load.append(REGION_COLUMN)
```

Reply:
This was motivated by the other suggestion. Without that, either variable name seems fine.

Reply:
Updated in [force-push](https://github.com/nextstrain/augur/compare/20d9e21b11016cbbbdf274081669ebfbf9e26cb1..455bcfab3a46c41ed69de0d9921967ce63d11d1a)